src/plugins/footnotes/index.ts

```diff
@@ -278,6 +335,48 @@ export default (props: PluginProps) => {
         }
       })
 
+      // unused footnotes go to the bottom of the list
+      footnotesRest.forEach(([node]) => footnotesReordered.push(node))
+
+      // replacing footnotes in footnote element
+      const newFElement = schema.nodes.footnotes_element.create(
+        footnoteElement[0].attrs,
+        footnotesReordered
+      )
+
+      if (newFElement) {
+        tr.replaceWith(
+          footnoteElement[1],
+          footnoteElement[1] + footnoteElement[0].nodeSize,
```

Comment:
Is this correct?

Reply:
It works. What seems to be off here?

Reply:
The indexes are different, is it expected?

Reply:
Understood.